lib/matplotlib/tests/test_mathtext.py

```diff
@@ -300,3 +301,34 @@ def test_mathtext_fallback_invalid():
 def test_mathtext_fallback_to_cm_invalid():
     for fallback in [True, False]:
         mpl.rcParams['mathtext.fallback_to_cm'] = fallback
+
+
+@image_comparison(['test_mathtext_fallback_cm'], extensions=['svg'])
```

Comment:
Instead of doing an image comparison (which is a rather indirect way to do the check), I would save the svg to a BytesIO() and then check that the correct substring (e.g. "font-family:STIXGeneral" or whatnot) appears in the buffer -- that directly tests the desired property.

Reply:
makes sense